File: Create&Explore.scala
// Databricks notebook source
dbutils.fs.ls("./")

// COMMAND ----------

dbutils.fs.mkdirs("/mnt/dados")

// COMMAND ----------

// MAGIC %python
// MAGIC display(dbutils.fs.ls("/mnt"))

// COMMAND ----------

val configs = Map(
  "fs.azure.account.auth.type" -> "OAuth",
  "fs.azure.account.oauth.provider.type" -> "org.apache.hadoop.fs.azurebfs.oauth2.ClientCredsTokenProvider",
  "fs.azure.account.oauth2.client.id" -> "*",
  "fs.azure.account.oauth2.client.secret" -> "*",
  "fs.azure.account.oauth2.client.endpoint" -> "https://login.microsoftonline.com/eca496fb-0e8f-4630-80ab-7319a08eca6d/oauth2/token")
dbutils.fs.mount(
  source = "link",
  mountPoint = "/mnt/dados",
  extraConfigs = configs)

File: Create&Explore.py
# Databricks notebook source
# MAGIC %python
# MAGIC configs = {"fs.azure.account.auth.type": "OAuth",
# MAGIC        "fs.azure.account.oauth.provider.type": "org.apache.hadoop.fs.azurebfs.oauth2.ClientCredsTokenProvider",
# MAGIC        "fs.azure.account.oauth2.client.id": "3e2d2549-4408-4aab-90a0-ef6f1d52c04f",
# MAGIC        "fs.azure.account.oauth2.client.secret": "tFX8Q~LVnBOGoTVy6tnDVi~wpjb0v~NdNvDGIaF3",
# MAGIC        "fs.azure.account.oauth2.client.endpoint": "https://login.microsoftonline.com/1a463d53-e4dc-4952-bf3c-6bac1d376e2d/oauth2/token",
# MAGIC        "fs.azure.createRemoteFileSystemDuringInitialization": "true"}
# MAGIC
# MAGIC dbutils.fs.mount(
# MAGIC     source = "abfss://datalake@basealura.dfs.core.windows.net/",
# MAGIC     mount_point = "/mnt/datalake/data",
# MAGIC     extra_configs = configs)
# MAGIC

# COMMAND ----------

# MAGIC %python
# MAGIC display(dbutils.fs.ls("/mnt/data/"))

# COMMAND ----------


